### PR TITLE
[14.0][MIG] product_multi_category

### PR DESCRIPTION
**Pull request by watthanun**

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Functional review only with runbot

image

File: product_multi_category/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2009 Akretion,Guewen Baconnier,Camptocamp,Avanzosc,Sharoon Thomas,Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Product - Many Categories",
+    "version": "14.0.1.0.0",
```
Comment (MeetKD) [inline product_multi_category/__manifest__.py:6]: @watthanun Summary key is missing.

File: product_multi_category/models/product.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    categ_ids = fields.Many2many(
+        comodel_name="product.category",
+        relation="product_categ_rel",
+        column1="product_id",
+        column2="categ_id",
+        string="Extra Categories",
```
Comment (MeetKD) [inline product_multi_category/models/product.py:15]: Instead of extra categories my suggestion is to change the string to multi categories.

**Review by MeetKD — COMMENTED** [review]
Minor Change + Suggestion.

**Review by joao-p-marques — APPROVED** [review]

File: product_multi_category/models/product.py (hunk 2 above)
Comment (joao-p-marques) [reply product_multi_category/models/product.py:15]: IMHO that would make sense if we just had a single field for all the categories. As they are separated between `Product Category` and  `Extra Categories`, I'd say it makes sense to keep it this way.

File: product_multi_category/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply product_multi_category/__manifest__.py:6]: Not really needed.

**Pull request merged**